src/containers/Tablet/TabletTable/TabletTable.tsx

```diff
@@ -33,6 +33,33 @@ const columns: Column<ITabletPreparedHistoryItem>[] = [
             return row.leader ? 'leader' : row.followerId;
         },
     },
+    {
+        name: 'Node ID',
+        align: DataTable.RIGHT,
+        sortable: false,
+        render: ({row}) => {
+            return <InternalLink to={getDefaultNodePath(row.nodeId)}>{row.nodeId}</InternalLink>;
```

Comment:
Should we have two similar links `Node ID` and `Node FQDN`?

Reply:
Discussed on call, actually there should be only links from Node IDs, as nodes with different IDs could be on the same host. I'll fix it